runtime/commonMain/src/kotlinx/serialization/json/JsonOutput.kt

```diff
@@ -51,3 +51,6 @@ public interface JsonOutput: Encoder, CompositeEncoder {
      */
     public fun encodeJson(element: JsonElement)
 }
+
+internal fun Encoder.asJsonOutput() = this as? JsonOutput
+        ?: throw SerializationException("This transformation can be used only with Json")
```

Comment:
Message assumes that it is used only from the transformer. Either the method should belong to or not to mention transformation.
Please use `JsonException` instead and be a bit more specific in the message: "Expected to have JsonOutput, but has this::class instead"

Reply:
Same for `asJsonInput`